versioned/tiered/rocksdb/src/main/java/org/projectnessie/versioned/rocksdb/Function.java

```diff
@@ -0,0 +1,85 @@
+/*
+ * Copyright (C) 2020 Dremio
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ * http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+package org.projectnessie.versioned.rocksdb;
+
+import java.util.Objects;
+
+import org.immutables.value.Value.Immutable;
+import org.projectnessie.versioned.impl.condition.ExpressionPath;
+import org.projectnessie.versioned.store.Entity;
+
+/**
+ * An expression that is asserted against an Entity.
+ */
+@Immutable
+abstract class Function {
+  static final String EQUALS = "equals";
```

Comment:
I thought @tifflhl changed this into an enum?

Reply:
She did but that patch didn't get in before I did the refactor. Should I apply her change to this branch?

Reply:
I'm applying this change.